components/tidb_query/src/rpn_expr/impl_cast.rs

```diff
@@ -179,7 +206,12 @@ fn in_union(implicit_args: &[ScalarValue]) -> bool {
     implicit_args.get(0) == Some(&ScalarValue::Int(Some(1)))
 }
 
-/// The unsigned int implementation for push down signature `CastIntAsDecimal`.
+fn is_field_type_unsigned(ft: &FieldType) -> bool {
```

Comment:
We can check whether is there `UNSIGNED` flag by `ft.as_accessor().is_unsigned()`, so I think we don't need this utility function.